signer/src/message.rs

```diff
@@ -72,7 +84,7 @@ pub struct SignerDepositDecision {
     pub accepted: bool,
 }
 
-/// Represents a decision related to signer withdrawal
+/// Represents a decision related to signer withdrawal.
```

Comment:
Couldn't decide if we prefer periods for shorter comments or not in my previous commits. :) Guess we prefer them now?

Reply:
I'm not even sure I would notice one way or the other. I'd go with whatever you feel like.

Reply:
Yeah I'm also unsure, but it feels slightly better to end sentences with a period - even for shorter comments so I'm trying to make that a habit - but as you can see from my previous PRs, I'm very inconsistent with that :sweat_smile: